.github/CONTRIBUTING.md

```diff
@@ -1,29 +1,15 @@
 # Contributing
 
-We'd love to accept your contributions to this project! There are just a few guidelines you need to follow.
-
-## Bugs
-
-Bug reports should be opened up as [issues](https://help.github.com/en/github/managing-your-work-on-github/about-issues) on the [go-vela/community](https://github.com/go-vela/community) repository!
-
-## Feature Requests
-
-Feature Requests should be opened up as [issues](https://help.github.com/en/github/managing-your-work-on-github/about-issues) on the [go-vela/community](https://github.com/go-vela/community) repository!
-
-## Pull Requests
-
-**NOTE: We recommend you start by opening a new issue describing the bug or feature you're intending to fix. Even if you think it's relatively minor, it's helpful to know what people are working on.**
-
-We are always open to new PRs! You can follow the below guide for learning how you can contribute to the project!
-
 ## Getting Started
 
+We'd love to accept your contributions to this project! If you are a first time contributor, please review our [Contributing Guidelines](https://go-vela.github.io/docs/community/contributing_guidelines/) before proceeding.
+
 ### Prerequisites
 
 - [Review the commit guide we follow](https://chris.beams.io/posts/git-commit/#seven-rules) - ensure your commits follow our standards
+- Review our [style guide](https://go-vela.github.io/docs/community/contributing_guidelines/#style-guide) to ensure your code is clean and consistent.
 - [Docker](https://docs.docker.com/install/) - building block for local development
 - [Docker Compose](https://docs.docker.com/compose/install/) - start up local development
-- [Golang](https://golang.org/dl/) - for source code and [dependency management](https://github.com/golang/go/wiki/Modules)
```

Comment:
Are you sure we want to remove Golang as a prerequisite?

If so, could you elaborate on why it's being removed from the list?

Reply:
I removed it because we address it in the [style guide ](https://go-vela.github.io/docs/community/contributing_guidelines/#style-guide)